src/llm/reason/prompt.py

```diff
@@ -46,6 +46,33 @@
 {related_past_episodes}
 
 
+[YOUR TASK]
+You are given the following task:
+{task}
+
+[TOOLS USAGE]
+You can ONLY ONE TOOL at a time and only tool that are listed below. Remember to use task_complete tool to mark the task as done.
```

Comment:
Adjust the prompt to:
You can ONLY USE ONE TOOL at a time and only use tools that are listed below